static/app/views/performance/transactionSummary/transactionTags/tagValueTable.tsx

```diff
@@ -152,7 +89,7 @@ export class TagValueTable extends Component<Props, State> {
         align={align}
         title={columnInfo.name}
         direction={currentSortKind}
-        canSort={false}
+        canSort
```

Comment:
Was this intentional? The `onClick` handler isn't implemented so this just adds the arrow without the ability to actually sort.

Reply:
Yeah it is actually sortable, sorting is done via the `generateSortLink` not the `onClick` lol, I'll remove the misleading comment in a follow up PR since I want to get this in.